appendix/contributing.md

```diff
@@ -0,0 +1,29 @@
+Contributing
+---------------------------
+
+Prerequisites
+---------------------------
+
+Reviewing the following will prepare you for contributing
+
+* You need to be comfortable with git, GitHub workflow of fork, branch, commit, open pull request, reviewing changes, and merge to work effectively in the KubeVirt community.
+* You need to be familiar with writing code in golang.  See the [golang tour](https://tour.golang.org/welcome/1) to familiarize yourself.
```

Comment:
This is if you want to contribute code... there are other ways to contribute (e.g. documentation, blog posts, raising issues, helping triage/reproduce...).

Maybe worth expanding / clarifying along those lines. You do talk about _other ways to contribute_ below already... point there?